api/src/main/java/data/Drills/Answer.java

```diff
@@ -0,0 +1,32 @@
+package data.Drills;
```

Comment:
Chtělo by to ještě zjistit, kam se v java aplikacich standatně dávají entity, ale myslím že něco jako `com.fprochazka.drill.model` bude dávat větší smysl.
Hlavně imho nechcem nic dávat do globálních packages mimo ten náš.